cmd/meshctl/templates/python/basic/Dockerfile.tmpl

```
@@ -3,15 +3,22 @@ FROM mcpmesh/python-runtime:0.7
 
-# Copy agent source code
-COPY . .
+# Copy agent source code and set permissions
+COPY --chmod=755 . .
+RUN chown -R mcp-mesh:mcp-mesh /app
+
+# Switch back to non-root user for security
+USER mcp-mesh
 
 # Expose the agent port
 EXPOSE {{ .Port }}
```

Hadolint (2.14.0)

[error] unexpected '{'
expecting '\', A port optionally followed by the protocol, A range of ports optionally followed by the protocol, a new line followed by the next instruction, or at least one space

(DL1000)